port/common/omrsysinfo.c

```diff
@@ -960,7 +960,7 @@ omrsysinfo_get_cgroup_subsystem_list(struct OMRPortLibrary *portLibrary)
  * @return TRUE if Runtime is running in a container and FALSE if not or if an error occurs
```

Comment:
```c
 * @param[in] errorCode int32_t pointer to state error code from internal calls
```
Needs to be removed as well

Reply:
Removed description for `errorCode` param.